lib/db/mysql/index.js

```diff
@@ -153,7 +153,7 @@ MysqlStore.prototype = {
   addAvatar: function addAvatar(id, uid, url, provider, selected) {
     id = buf(id);
     uid = buf(uid);
-    var store = this;
+    var store = this; // eslint-disable-line consistent-this
```

Comment:
Huh, what does eslint want me to name `this`? I don't follow the crowd and rename to a single thing, but usually rename to what `this` is conceptually...